src/plus/gk/account/promos.ts

```diff
@@ -21,6 +21,7 @@ const promos: Promo[] = [
 			SubscriptionState.FreePlusInTrial,
 			SubscriptionState.FreePlusTrialExpired,
 			SubscriptionState.FreePlusTrialReactivationEligible,
+			SubscriptionState.PaidExpired,
```

Comment:
We probably want to include this state on the `pro50` promo as well.

Reply:
**Edit:** I was wrong - as you mentioned, the promo is only for first seat: https://github.com/gitkraken/vscode-gitlens/pull/3522#discussion_r1758517486